python/tests/artm/test_batches_utils.py

```diff
@@ -81,11 +105,9 @@ def test_func():
                     
             assert counter == n_wd_num_tokens + 2
 
-            # The following tests are commented out because they don't pass with Python 3
-            # Please, investigate.
-            # assert tokens == n_wd_tokens_list
-            # assert token_tf == n_wd_token_tf_list
-            # assert token_df == n_wd_token_df_list
+            assert set(tokens) == set(n_wd_tokens_list)
+            assert set(token_tf) == set(n_wd_token_tf_list)
+            assert set(token_df) == set(n_wd_token_df_list)
```

Comment:
@MelLain Could you provide some brief results of investigation (i.e why we can't compare lists in Python3)?